Description:
 01_fix_static_datadir_evaluation.dpatch by Daniel Leidert (dale) <daniel.leidert@wgdd.de>
 All lines beginning with `## DP:' are a description of the patch.
 The evaluation of datadir results in "${prefix}/share" without
 evaluation of the ${prefix} variable with autoconf 2.60.

Index: docbook2X-0.8.8/configure.ac
===================================================================
--- docbook2X-0.8.8.orig/configure.ac
+++ docbook2X-0.8.8/configure.ac
@@ -148,7 +148,7 @@
 dnl they will reside and should use these static_* values.
 dnl Ensure that all static_* are fully expanded.
 
-eval static_datadir="$datadir"
+eval eval static_datadir="$datadir"
 
 eval static_bindir="$bindir"
 old_val=""
